packages/shopware-6-client/src/endpoints.ts

```diff
@@ -1 +1,79 @@
-export const CATEGORY_ENDPOINT = "/category";
+// category
+
+export const CATEGORY_ENDPOINT = () => `/category`;
```

Comment:
yeah, in fact, the name suggests that this is a const, not method
les's change these to a more method-conventional name like `getXxxEndpoint`
```suggestion
export const getCategoryEndpoint = () => `/category`;
```